README.md

```diff
@@ -94,3 +94,11 @@ Finally, run `jupyter lab` to open a Jupyter server in the TCV-X21 repository. T
 
 To ensure that the results are reproducible, the `environment.yml` file has pinned dependencies. However, if you want to use
 this software as a library, pinned dependencies are unnecessarily restrictive. You can remove the versions after the `=` sign in the `environment.yml`, but be warned that things might break.
+
+### Acknowledgement guidelines
+
+If you use any of the content of the TCV-X21 repository in one of your publications and/or talks, please cite (DOI`s link?) and properly place the acknowledgements as instructed in bellow. We would also appreciate it if you informed us about your work by sending us an email to spc.data@epfl.ch. This will let us keep track of how the dataset is being used. 
```

Comment:
Correct DOI is [10.1088/1741-4326/ac4cde](doi.org/10.1088/1741-4326/ac4cde)

You can add links in Markdown with `[link text](url)`